client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
-			title: 'Child Themes',
-			description:
+			title: translate( 'Child Themes' ),
+			description: translate(
 				"The only limit on your site is your vision — if the themes you see don't match that, it's " +
-				'time to go beyond them. Learn to use child themes to customize and extend your website.',
+					'time to go beyond them. Learn to use child themes to customize and extend your website.'
+			),
 		},
 	],
 	theme: [
 		{
 			link: 'http://en.support.wordpress.com/themes/',
-			title: 'Themes: An Overview',
-			description:
+			title: translate( 'Themes: An Overview' ),
+			description: translate(
 				'A theme controls the general look and feel of your site including things like ' +
-				'page layout, widget locations, and default font.',
+					'page layout, widget locations, and default font.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/themes/mobile-themes/',
-			title: 'Mobile Themes',
-			description:
+			title: translate( 'Mobile Themes' ),
+			description: translate(
 				'When a visitor browses to a WordPress.com site on a mobile device, we show ' +
-				'special themes designed to work on small screens focusing on fast load times.',
+					'special themes designed to work on small screens focusing on fast load times.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/premium-themes/',
-			title: 'Premium Themes',
-			description:
+			title: translate( 'Premium Themes' ),
+			description: translate(
 				'On a site with the Premium or Business plan, you can switch to any premium theme at ' +
-				'no extra cost, as many times as you’d like.',
+					'no extra cost, as many times as you’d like.'
+			),
 		},
 		{
 			link:
 				'http://en.support.wordpress.com/themes/uploading-setting-up-custom-themes/child-themes/',
-			title: 'Child Themes',
-			description:
+			title: translate( 'Child Themes' ),
+			description: translate(
 				"The only limit on your site is your vision — if the themes you see don't match that, it's " +
-				'time to go beyond them. Learn to use child themes to customize and extend your website.',
+					'time to go beyond them. Learn to use child themes to customize and extend your website.'
+			),
 		},
 	],
 	plans: [
 		{
 			link: 'http://en.support.wordpress.com/discover-the-wordpress-com-plans/',
-			title: 'Explore the WordPress.com Plans',
-			description:
-				"Upgrading your plan unlocks a ton of features! We'll help you pick the best fit for your needs and goals.",
+			title: translate( 'Explore the WordPress.com Plans' ),
+			description: translate(
+				"Upgrading your plan unlocks a ton of features! We'll help you pick the best fit for your needs and goals."
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/plan-features/',
 			post_id: 134698,
-			title: 'WordPress.com Plans',
-			description:
-				'Learn about the capabilities and features that the different plans unlock for your site.',
+			title: translate( 'WordPress.com Plans' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Discover WordPress.com Plans' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).